automation/check-patch.e2e-k8s.sh

```diff
@@ -6,7 +6,6 @@ teardown() {
 }
 
 main() {
-    export KUBEVIRT_PROVIDER='k8s-1.17.0'
```

Comment:
This should stay. The lane is explicitly pinned to a certain version. Using the default from another module is implicit.